test/cli.js

```diff
@@ -364,6 +364,25 @@ test('bails when config contains `"tap": true` and `"watch": true`', t => {
 	});
 });
 
+['--concurrency', '-c'].forEach(concurrencyFlag => {
+	test(`bails when ${concurrencyFlag} provided without value`, t => {
+		execCli([concurrencyFlag, 'test.js'], {dirname: 'fixture/concurrency'}, (err, stdout, stderr) => {
+			t.is(err.code, 1);
+			t.match(stderr, 'The --concurrency and -c flags must be provided the maximum number of test files to run at once.');
```

Comment:
What does `stderr` actually contain?

I had some trouble running this directly from the command line, it seemed to start running TAP tests instead. Try `execCli(['test.js', concurrencyFlag], …)` instead.